kore/src/Kore/Internal/TermLike.hs

```diff
@@ -1557,6 +1558,14 @@ mkDefined = updateCallStack . worker
 mkDefinedAtTop :: Ord variable => TermLike variable -> TermLike variable
 mkDefinedAtTop = synthesize . DefinedF . Defined
 
+unDefined :: TermLike variable -> TermLike variable
```

Comment:
Maybe it would be useful to document this as well, as we have `mkDefinedAtTop` and `mkDefined`?